govcd/vapp_network.go

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2020 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
+
+package govcd
+
+import (
+	"fmt"
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+	"github.com/vmware/go-vcloud-director/v2/util"
+	"net/http"
+)
+
+// UpdateNetworkFirewallRules updates vApp networks firewall rules.
+// Returns pointer to types.VAppNetwork or error
```

Comment:
I think this comment (and the "async" version) deserves a note that all rules are overwritten every time.

Reply:
Added